storm-core/src/clj/backtype/storm/daemon/supervisor.clj

```diff
@@ -675,6 +675,7 @@
                         (add-to-classpath topo-classpath))
           top-gc-opts (storm-conf TOPOLOGY-WORKER-GC-CHILDOPTS)
           gc-opts (substitute-childopts (if top-gc-opts top-gc-opts (conf WORKER-GC-CHILDOPTS)) worker-id storm-id port)
+          topo-worker-logwriter-childopts (conf TOPOLOGY-WORKER-LOGWRITER-CHILDOPTS)
```

Comment:
Good catch. Since user may have configuration for this in topology's stormconf.ser. Updated.